Initialize project (#1)
werkraum-media/form_file_collection: README.rst

====================================
TYPO3 Extension Form File Collection
====================================

Adds new EXT:form element used to render a file collection allowing the visitor to choose files.

Concept
=======

The form element will fetch the configured file collection and assign proper options based on the contained files.

> That allows for using existing templates that allow to select one or multiple options.

Reviewer: Sounds weird. Maybe:

```suggestion
That allows usage of existing templates that already handle one or multiple options.
```
